src/sasjs-job/execute.js

```diff
@@ -0,0 +1,93 @@
+import chalk from 'chalk'
+import ora from 'ora'
+import { displayResult } from '../utils/displayResult'
+
+/**
+ * Triggers existing job for execution.
+ * @param {object} sasjs - configuration object of SAS adapter.
+ * @param {string} accessToken - an access token for an authorized user.
+ * @param {string} jobPath - location of job at SAS server.
+ * @param {object} target - SAS server configuration.
+ * @param {boolean} waitForJob - flag indicating if CLI should wait for job completion.
+ * @param {boolean} showOutput = flag indicating if CLI should print out job output.
```

Comment:
`@param {boolean} log - filepath to which to write the job log.  Will only be returned if waitForJob is true.